index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+     
+         <div class="header">
+               <div class="logo">
+                   <h1>My Logo</h1>
+               </div>
+               <div class="navbar-item">
+                   <a href="#">Portfolio</a>
+                   <a href="#">About</a>
+                   <a href="#">contact</a>
+                   <a href=""><img src="/images/Rec.png" alt=""></a>
+                   <img src="/images/Union.png" alt="">
+                   <img class="icon" src="/images/Union1.png" alt="">
+               </div>
+         </div>
```

Comment:
- [ ] Please I highly recommend changing `div` tag to `header` tag to implement HTML semantics for better user experience. Please [check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more information. 😄